docs/components/compact.md

```diff
@@ -28,6 +28,18 @@ config:
 The compactor needs local disk space to store intermediate data for its processing. Generally, about 100GB are recommended for it to keep working as the compacted time ranges grow over time.
 On-disk data is safe to delete between restarts and should be the first attempt to get crash-looping compactors unstuck.
 
+# Resolution and retention
+
+Resolution - distance between data points on your graphs. E.g.
+
+raw - the same as scrape interval at the moment of data ingestion
+5m - data point is every 5 minutes
+1h - data point is every 1h
+
+Keep in mind, that the initial goal of downsampling is not saving a disk space. The goal of downsampling is providing an opportunity to get fast approximate results for range queries of big time intervals like months or years. In other words, if you set `--retention.resolution-raw` less then `--retention.resolution-5m` and `--retention.resolution-1h` - you might run into a problem of not being able to "zoom in" after you queried your data for 1 year interval.
```

Comment:
Nice! Well put, some nits though:

* I would remove `approximate`. We made sure the data are mathematically correct (: So it should be correct.
* I removed ` after you queried your data for 1 year interval.` as I found this bit confusing -> you can zoom in even before I query long range query (: 

```suggestion
Keep in mind, that the initial goal of downsampling is not saving disk space. The goal of downsampling is providing an opportunity to get fast results for range queries of big time intervals like months or years. In other words, if you set `--retention.resolution-raw` less then `--retention.resolution-5m` and `--retention.resolution-1h` - you might run into a problem of not being able to "zoom in" to your historical data.
```

I would also add something about Querier being able to automatically deduce what resolution you need based on query `step` (for now). 

What do you think?

Reply:
Also, can we add something about expected size:

```
In fact, downsampling adds 2 more blocks for each raw block which are only slightly smaller or relatively similar size to raw block. This is required by internal downsampling implementation which to be mathematically correct holds various aggregations. This means that downsampling can increase the size of your storage a bit (~3x), but it gives massive advantage on querying long ranges. 
```

Reply:
@bwplotka I'd also elaborate on the exact list of aggregation so users would know what might be effectively used as a pre-calculated dataset.

Can you point me to where I can find such a list?